init.rb

```diff
@@ -5,7 +5,7 @@
   version '1.0.4'
   url 'https://github.com/clear-code/redmine_full_text_search'
   author_url 'https://github.com/clear-code'
-  directory __dir__
+  directory File.dirname(File.absolute_path(__FILE__))
```

Comment:
Could you add a comment with example why we can't use `__dir__` here?

For example:

```ruby
# We can't use __dir__ here because ...
#
# For example, _dir__ and __FILE__ are the followings when we use symbolic link:
#   * __dir__: ...
#   * __FILE__: ... 
```

Reply:
fix: [e008ca9](https://github.com/clear-code/redmine_full_text_search/pull/129/commits/e008ca95e06b0872d1f7bf7516e2b6f931403472) Sure. I understood we should add the code comment because users cannot understand why we use `__FILE__` instead of `__dir__` from the code.